### PR TITLE
[6.3][Cherry-Pick]Add sanity test for navigation functionality and cover BZ1394974

### PR DESCRIPTION
**Pull request by oshtaier**
Same as https://github.com/SatelliteQE/robottelo/pull/4283
Situation with defects is completely different here as issue in 6.2 is resolved in 6.3, but introduced a new one
```
nosetests tests/foreman/ui/test_navigation.py
S.
----------------------------------------------------------------------
Ran 2 tests in 50.469s

OK (SKIP=1)
```

### CODE REVIEW

File: tests/foreman/ui/test_navigation.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+from robottelo.decorators import skip_if_bug_open, tier1
+from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, menu_locators
+from robottelo.ui.session import Session
+
+
+class NavigationTestCase(UITestCase):
+    """Implements the navigation tests from UI"""
+
+    def page_objects(self):
```
Comment (rochacbruno) [inline tests/foreman/ui/test_navigation.py]: Could be a property?

```python
@property
def page_objects(self):
```

then

```python
for page in self.page_objects.values():  # without the ( )
```

Or renamed to `get_page_objects` to be used as a method `for page in self.get_page_objects()`

?

File: tests/foreman/ui/test_navigation.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        @BZ: 1394974
+
+        @Assert: No error is raised after redirection and logo is still present
+        on the page
+        """
+        with Session(self.browser) as session:
+            pages = self.page_objects()
+            for page_name in (
+                    'Activation Key', 'Product', 'Domain', 'Location'):
+                pages[page_name].navigate_to_entity()
```
Comment (rochacbruno) [inline tests/foreman/ui/test_navigation.py]: if a property is defined you can simply do:

```python
self.page_objects[page_name].navigate_to_entity()
```

**Comment by oshtaier** [comment]
@rochacbruno Agree with you that it is better approach, but don't want to have different code in different branches. Let me know whether it is critical to fix code in that way, so I will work on it for both branches

**Comment by rochacbruno** [comment]
@oshtaier ok, we can change it later..

**Pull request merged**